6001-7000/LIT6528MartyrdomAbrahamGarga.xml

```diff
@@ -51,6 +51,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <revisionDesc>
             <change who="ES" when="2021-12-29">Created entity</change>
             <change who="MKr" when="2024-11-08">Added relations</change>
+            <change who="MKr" when="2024-11-27" >Corrected relation</change>
```

Comment:
Delete space inside the xml-string: `<change who="MKr" when="2024-11-27">Corrected relation</change>`